N/NeuralNetDiffEq/Deps.toml

```diff
@@ -2,3 +2,7 @@
 DiffEqBase = "2b5f629d-d688-5b77-993f-72d75c75574e"
 Flux = "587475ba-b771-5e3f-ad9e-33799f191a9c"
 Reexport = "189a3867-3050-52da-a836-e630ba90ab69"
+
+["0.1-0"]
+SafeTestsets = "1bc83da4-3b8d-516f-aca4-4fe02f6d838f"
```

Comment:
This addition looks unintentional, given that you only use it in tests and also list it under extras: https://github.com/JuliaDiffEq/NeuralNetDiffEq.jl/blob/190903f6648e32c3f3457a59acde4c7b97c7038e/Project.toml#L19

Reply:
Yup it was a mistake. Fixed